code/frameworks/angular/src/builders/utils/run-compodoc.ts

```diff
@@ -28,7 +28,7 @@ export const runCompodoc = (
     const packageManager = JsPackageManagerFactory.getPackageManager();
 
     try {
-      const stdout = packageManager.runPackageCommand(
+      const stdout = packageManager.runPackageCommandSync(
```

Comment:
I had to create a `runPackageCommandSync` just because of this code 😓

@valentinpalkovic do you think this could be turned into async?

Reply:
Yes! It also had to be turned into async, as soon as we support watch mode. I will do this in a follow up PR